examples/ansible.cfg

```diff
@@ -278,7 +278,7 @@
 # choice but to create world readable temporary files to execute a module on
 # the remote machine. This option is False by default for security. Users may
 # turn this on to have behaviour more like Ansible prior to 2.1.x. See
-# https://docs.ansible.com/ansible/latest/user_guide/become.html#becoming-an-unprivileged-user
+# https://docs.ansible.com/ansible/latest/user_guide/become.html#risks-of-becoming-an-unprivileged-user
```

Comment:
'latest' will eventually be wrong, it should keep either within it's version (2.11) or be relative (why it should use doclink)

Reply:
As long as we leave it hard-coded to /ansible/latest, I think this is the best we can do. There are no docs at the URL /ansible/2.11/. At any rate, I didn't change that part in this PR - if we want to fix it to use relative links we can do that in another PR. Meanwhile at least this will take users to a relevant section of the docs.

Reply:
Also, the advice on handling the risks of becoming an unprivileged user have not changed, nor are they likely to.